Gemfile

```diff
@@ -11,7 +11,7 @@ gem 'jar-dependencies', '~> 0.3.11', :require => nil
 gem 'mocha', '~> 1.4', '< 2.0'
 
 # for the rake task
-gem 'ruby-maven', github: 'deivid-rodriguez/ruby-maven', branch: 'exit_status'
+gem 'ruby-maven', github: 'deivid-rodriguez/ruby-maven', branch: 'master'
```

Comment:
please rebase, this was due https://github.com/jruby/ruby-maven/pull/1 which is merged upstream now

Reply:
Done